less_slow.cpp

```diff
@@ -2245,6 +2316,9 @@ BENCHMARK(cblas_tops<double>)->RangeMultiplier(2)->Range(8, 65536)->Complexity(b
 
 template <typename scalar_type_>
 static void eigen_tops(bm::State &state) {
+    // Make sure Eigen uses all cores - also, Eigen can't multithread without openMP or GEMM
```

Comment:
What's GEMM?

Reply:
General Matrix-Matrix product, I guess the more correct comment would be to say the internal GEMM threadpool that Eigen manages. There seems to be a way to make eigen use that instead of openMP (there is a EIGEN_GEMM_THREADPOOL flag that is checked) but I'm not too familiar with eigen, so not sure how that would be done.